instance/models/mixins/openedx_storage.py

```diff
@@ -61,7 +61,10 @@ def get_storage_settings(self):
         Only needed when not using ephemeral databases
```

Comment:
@pomegranited Removed. I also changed the EDXAPP_IMPORT_EXPORT_BUCKET setting to an empty string to be consistent with the [change we made to the configuration repo](https://github.com/edx/configuration/pull/3906).